public/docs/ts/latest/guide/i18n.jade

```diff
@@ -0,0 +1,261 @@
+  Those finished translations can then be loaded alongside with your application to make it
+  available to multiple audiences.
+  
+  <a id="top"></a>
+  ## Table of contents
+    
+    * [Angular and i18n](#angular-i18n)
+    * [The i18n attribute](#the-i18n-attribute)
+    * [Extract messages with ng-xi18n](#ng-xi18n)
+    * [Translate messages](#translate)
+    * [Loading translations](#loading-translations)
+      * [Using JiT](#use-jit)
+      * [Using AoT](#use-aot)
+
+:marked
+  **Try it out**. Here's a link to a <live-example></live-example> of the JiT application.      
+
+  
+a(id="angular-i18n")
+.l-main-section
+:marked
+  ## Angular and i18n
+
+  Angular allows us to replace text in our application with the translation of our choice. 
+  
+  We would usually start with a working application that already has text in a single language. 
+  We then we extract from our app a translation file that translators can load on their translation software to work on the translation.
+  Finally we use their translation on our app to override the original text.
+
+  We can then build the application for each language we need, and deploy each translated application to it's own server.  
+
+  
+a(id="the-i18n-attribute")
+.l-main-section
+:marked
+  ### i18n attribute
+  
+  The `i18n` attribute is what tells Angular where it should insert the translations.
+
+  We'll start with a simple greeting in a `<h1>` tag:
+
++makeExample('i18n/ts/app/app.component.1.html', 'greeting', 'app/app.component.html')(format=".")
+
+:marked
+  We simply add the attribute `i18n` to our tag to mark it as a translation point.
+
++makeExample('i18n/ts/app/app.component.1.html', 'i18n-attribute', 'app/app.component.html')(format=".")
+
+:marked
+  To help the translators, we can add more information about the meaning and context of this string.
+  Simply add a description to the i18n attribute:
+  
++makeExample('i18n/ts/app/app.component.1.html', 'i18n-attribute-desc', 'app/app.component.html')(format=".")
+
+:marked
+  We can add some meaning as well, separate the meaning from the description with the `|` character:
+  
++makeExample('i18n/ts/app/app.component.html', 'i18n-attribute-meaning', 'app/app.component.html')(format=".")
+
+:marked
+  While the same text with the same meaning should have the same translation, 
+  the same text with *different meanings* can have different translations.
+  Both the meaning and the description will be extracted by our messages extractor and added to the messages file.
+  
+  This will help our translators to translate our application with a better understanding of what our text means.   
+
+
+a(id="ng-xi18n")
+.l-main-section
+:marked
+  ### Extract messages with ng-xi18n
+  
+  Now that our template has been updated to support i18n translations, we can use the `ng-xi18n` messages extractor.
+  
+  This CLI tool is based on `ngc`, and is available in the `@angular/compiler-cli` npm package.
```

Comment:
I think it's relevant, since it'll use the `ngc` options. For instance, it'll use genDir.